frontend/pages/home_page.js

```diff
@@ -1,34 +1,73 @@
 import React, { Component, PureComponent } from 'react';
 import { Redirect } from 'react-router';
 import appRoutes from 'src/constants/routes';
+import http from 'src/frontend/services/http';
 
 export default class HomePage extends Component {
   state = {
     loading: false,
     error: '',
     redirectTo: '',
+    products: '',
   }
 
   goToProductNewPage = () => {
     this.setState({ redirectTo: appRoutes.productNew });
   }
 
-  render() {
-    const { error, redirectTo } = this.state;
-    if (redirectTo) return <Redirect push to={redirectTo} />;
+  fetchProducts = () => {
+    http.get('/product').then(res => {
+        this.setState({
+          products: res.products
+        })
```

Comment:
missing semicolon, 2 spaces instead of 4, and if the line is short i like to keep 1 liners: 
```
this.setState({ products: res.products });
```